pkg/network/infraconfigurators/masquerade_test.go

```diff
@@ -462,11 +487,15 @@ func mockIPTablesBackend(handler *netdriver.MockNetworkHandler, proto iptables.P
 		"-j",
 		"KUBEVIRT_POSTINBOUND").Return(nil)
 
-	for _, chain := range []string{"OUTPUT", "KUBEVIRT_POSTINBOUND"} {
-		handler.EXPECT().IptablesAppendRule(proto, "nat", chain,
-			"-p", "tcp", "--match", "multiport",
-			"--dports", fmt.Sprintf("%s", strings.Join(PortsUsedByLiveMigration(), ",")),
-			"--source", GetLoopbackAdrress(proto), "-j", "RETURN").Return(nil)
+	skipPorts := portsUsedByLiveMigration(isMigrationOverSockets)
+
+	if len(skipPorts) > 0 {
```

Comment:
Limiting the scope of `skipPorts` may be useful to be clear it is not used later on.
`if skipPorts := portsUsedByLiveMigration(isMigrationOverSockets); len(skipPOrts) > 0 {`